arch/riscv/Kconfig.isa

```diff
@@ -111,6 +113,20 @@ config RISCV_ISA_EXT_ZIFENCEI
 	  provides explicit synchronization between writes to instruction
 	  memory and instruction fetches on the same hart.
 
+config RISCV_ISA_EXT_ZAAMO
+	bool
+	help
+	  (Zaamo) - Atomic memory operation subset of the A extension
+
+	  The Zaamo extension enables support for AMO*.W/D-style instructions.
+
+config RISCV_ISA_EXT_ZLRSC
+	bool
+	help
+	  (Zlrsc) - Load-Reserved/Store-Conditional subset of the A extension
+
```

Comment:
```suggestion
```

Reply:
Ditto - this blank line was added to keep with the convention